WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
             ),
         },
-        "TTGamma_SingleLeptFromT" : {
-            "CrossSection" : XSValues(
-                XS_2016=0.7684, XSSource_2016="XSDB (NLO)",
-                XS_2017=0.7040, XSSource_2017="XSDB (LO)",
-                XS_2018=0.7053, XSSource_2018="GenXSecAnalyzer",
+        "TTTo2L2Nu_mtop171p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=706.1, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "TTGamma_SingleLeptFromTbar" : {
-            "CrossSection" : XSValues(
-                XS_2016=0.7659, XSSource_2016="XSDB (NLO)",
-                XS_2017=0.7040, XSSource_2017="XSDB (LO)",
-                XS_2018=0.7028, XSSource_2018="GenXSecAnalyzer",
+        "TTTo2L2Nu_mtop173p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=668.6, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "TTGamma_Dilept" : {
-            "CrossSection" : XSValues(
-                XS_2016=0.6343, XSSource_2016="XSDB (NLO)",
-                XS_2017=0.5795, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=0.5799, XSSource_2018="GenXSecAnalyzer",
+        "TTTo2L2Nu_mtop175p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=633.4, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-70To100" : {
-            "CrossSection" : XSValues(
-                XS_2016=1353.0, XSSource_2016="XSDB (LO)",
-                XS_2017=1292.0, XSSource_2017="XSDB (LO)",
-                XS_2018=1292.0, XSSource_2018="GenXSecAnalyzer",
-            ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+        "TTTo2L2Nu_mtop178p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=584.6, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu_HT-100To200" : {
+        "TTToHadronic" : {
             "CrossSection" : XSValues(
-                XS_2016=1346.0, XSSource_2016="XSDB (LO)",
-                XS_2017=1395.0, XSSource_2017="XSDB (LO)",
-                XS_2018=1393.0, XSSource_2018="GenXSecAnalyzer",
-            ),
-            "kFactor" : kFactorValues(
-                kFactor_2016=1.224, kFactorSource_2016="XSDB NNLO/LO=61526.7/50260",
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
-            ),
-            "Correction" : CorrValues(
-                Corr_2017=0.993, CorrSource_2017="https://twiki.cern.ch/twiki/bin/viewauth/CMS/MCKnownIssues#WJetsToLNu_HT_and_DYJets_HT_LO_M",
-                Corr_2018=0.993, CorrSource_2018="Same as 2017",
+                XS_13TeV=687.1, XSSource_13TeV="XSDB (NLO)",
```

Comment:
Again, looks like you're using an NLO and not the NNLO PDG value. Also, you appear to be missing the BR to all-hadronic (66.5% from https://pdg.lbl.gov/2018/listings/rpp2018-list-t-quark.pdf). Again, we could use the built-in BR mechanism to separate the cross sections and branching ratios so that this confusion doesn't happen again.